cupyx/scipy/spatial/distance.py

```diff
@@ -494,6 +494,37 @@ def cdist(XA, XB, metric='euclidean', out=None, **kwargs):
         raise TypeError('2nd argument metric must be a string identifier')
 
 
+def pdist(X, metric='euclidean', out=None, **kwargs):
```

Comment:
```suggestion
def pdist(X, metric='euclidean', *, out=None, **kwargs):
```

Reply:
We're taking care of making CuPy functions always have exactly the same signatures against NumPy/SciPy counterparts whenever possible.